playbook.yml

```diff
@@ -0,0 +1,191 @@
+- hosts: us
```

Comment:
this playbook is pretty straightforward, needs a bit of love as it's littered with the detritus of previous versions of everything, but it's all basic stuff. Doesn't (imo) need thorough review. Unless you think it's confusing in which case do tell!